service/distros.go

```diff
@@ -25,6 +25,16 @@ import (
 
 func (uis *UIServer) distrosPage(w http.ResponseWriter, r *http.Request) {
 	u := MustHaveUser(r)
+
+	flags, err := evergreen.GetServiceFlags(r.Context())
+	if err != nil {
+		gimlet.WriteResponse(w, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving admin settings")))
+	}
+	if flags.LegacyUIDistroPageDisabled {
+		newUIProjectsLink := fmt.Sprintf("%s/distros", uis.Settings.Ui.UIv2Url)
```

Comment:
We can remove this line and move `spruceLink` on L89 right before this block to prevent initializing the same value twice